apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -64,27 +64,24 @@ const ProjectAppTypeArea = React.createClass({
     // from redux dispatch
     appendProjects: PropTypes.func.isRequired,
     setHasOlderProjects: PropTypes.func.isRequired,
-  },
-
-  getInitialState() {
-    return {
-      maxNumProjects: this.props.projectList ? this.props.projectList.length : 0,
-      numProjects: this.props.numProjectsToShow,
+  };
 
-      // Disables the View More button when a network request is pending.
-      disableViewMore: false,
-    };
-  },
+  state = {
+    maxNumProjects: this.props.projectList ? this.props.projectList.length : 0,
```

Comment:
This isn't going to work - since the initial state depends on `this.props` you're going to have to define it in a constructor.